src/restconfig/src/test/java/org/geoserver/rest/catalog/CatalogRESTTestSupport.java

```diff
@@ -40,6 +41,8 @@ protected void onSetUp(SystemTestData testData) throws Exception {
 
         XMLUnit.setXpathNamespaceContext(new SimpleNamespaceContext(namespaces));
         xp = XMLUnit.newXpathEngine();
+        // attach time stamp listener
+        new CatalogTimeStampUpdater(catalog);
```

Comment:
This looks weird at a first look, why does the listener needs to be added again here?

Reply:
the listner has been declared universely inside the gs-man POM file, the tests did not pick it up, hence to make it work inside REST test I had to add it to see how things work with REST catalog calls.